include/fizzy/fizzy.h

```diff
@@ -33,10 +33,10 @@ typedef struct FizzyExecutionResult
     /// Whether execution ended with a trap.
     bool trapped;
     /// Whether function returned a value.
-    /// Equals false if trapped equals true.
+    /// Equals false if `trapped` equals true.
```

Comment:
```suggestion
    /// Equals false if #trapped equals true.
```
I'm not sure that the goal is here, but you can reference the class variable like this. See https://www.doxygen.nl/manual/autolink.html.